docs/datascience/README.md

```diff
@@ -125,6 +133,57 @@ If you want to use another language, we still recommend stating with the [tutori
 This is the easiest way to download the data and create new tomes, which are just more Parquet and JSON files.
 Tomes are economical views of the data tailored for specific analyses.
 
+## Cost FAQ
+
+### How much will AWS charge me?
+
+_FPS Critic Inc., owner of PureSkill.gg, is not liable for any AWS costs you incur._
+
+[Step 7 of the tutorial] can help you estimate your costs,
+We provide those and the estimates below for convenience,
+but we do not guarantee their accuracy or applicability to your AWS account.
+
+TL;DR: It costs about $1.50 to download a day of data, or $4-5 per 1,000 matches.
+Downloading tomes is much less expensive.
+
+- Tomes contain slices of features across many matches.
+
+### How should I optimize my costs?
+
+The data set is structured to help you control and optimize your costs.
+First, it is split into daily revisions which may be downloaded individually.
+Second, we maintain a companion tome data set which is very economical.
+
+Consider these steps:
+
+1. You probably don't need as much data as you think.
+2. AWS has a generous free tier which
+   may reduce or the eliminate the cost to get started.
+3. If you can use available tomes for your project,
+   don't bother downloading individual match data.
+   Most tomes cost pennies to download and contain
+   data for across many matches.
+4. Otherwise, start with one day of data to discover your costs.
+5. Estimate how much data you actually need to get started experimenting.
+6. Be careful when downloading a large number of revisions at once.
+   You might go over the free tier part-way though the export process
+   and have difficulty stopping the jobs.
+   We recommend you limit each export batch to one month intervals.
+7. If you share a tome generator with us, we may be able to make the tome
+   for you and share it back to the data set.
+
+### Where can I get financial support?
+
+- If you are a student, ask your department or advisor if they can support you.
+- If you are a professional data scientist, ask your organization
+  if they can fund this as a hackathon project or self-guided learning project.
+- Ask AWS for credits.
+  Your company, organization, university, or department may
+  have a relationship with AWS.
+  If not, talk to your AWS account manager and they may help you get some credits.
+- Talk to us. We can't promise financial support, but we can brainstorm a solution together.
```

Comment:
We could put the tomes onto drive or like dropbox for easier access (though both would cost money for us). Maybe some sort of peer-to-peer transfer is possible? I feel like this is the real use case for torrents but I have no experience with that.

Reply:
Yes. Same could be said for the whole dataset: one torrent per month, however I don't want to bypass the ADX just yet.